erpnext/e_commerce/product_search.js

```diff
@@ -0,0 +1,244 @@
+erpnext.ProductSearch = class {
+	constructor() {
+		this.MAX_RECENT_SEARCHES = 4;
+		this.searchBox = $("#search-box");
+
+		this.setupSearchDropDown();
+		this.bindSearchAction();
+	}
+
+	setupSearchDropDown() {
```

Comment:
```suggestion
	setup_search_dropdown() {
```
Why camel case in this file?

Reply:
Was following general JS case, wasn't sure if i should keep it consistent with the case in frappe since its an independent JS file. Will change it